packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -118,6 +124,15 @@ import {
 } from '../../../../stores/shared/ModelUtil.js';
 import { LEGEND_STUDIO_APPLICATION_NAVIGATION_CONTEXT_KEY } from '../../../../stores/LegendStudioApplicationNavigationContext.js';
 
+interface ClassPropertyDragSource {
+  columnEditorState: ClassEditorState;
+  columnProperty: ClassEditorState['selectedProperty'];
+}
+
+enum CLASS_PROPERTY_DND_TYPE {
+  PROJECTION_COLUMN = 'PROJECTION_COLUMN',
```

Comment:
Projection column refers to the column returns in a projection function. 
Probably mean PROPERTY?

Reply:
Ah yes I do thank you will switch